content/docs/specification/content.md

```diff
@@ -198,60 +240,63 @@ https://tools.ietf.org/html/rfc3986#page-11
 * Example: "to=/docs/"
 -->
 
-**Repository Type**  
-
-* Key: **vcs**
-* Recommended
-* Default: **git**
-* Permitted values: "git|bzr|fossil|hg|svn"
-* Example: "to=https://github.com/txtdirect/"
+**Repository Type**
 
+- Key: **vcs**
+- Recommended
+- Default: **git**
+- Permitted values: "git|bzr|fossil|hg|svn"
+- Example: "to=https://github.com/txtdirect/"
 
 ---
+
 # Proxy Type (experimental)
-**Type**  
 
-* Key: **type**
-* Mandatory
-* Permitted values: "proxy"
-* Example: "type=proxy"
+**Type**
 
-**Version**  
+- Key: **type**
+- Mandatory
+- Permitted values: "proxy"
+- Example: "type=proxy"
 
-* Key: **v**
-* Mandatory
-* Permitted values: "txtv0"
-* Example: "v=txtv0"
+**Version**
 
-**Upstream URL**  
+- Key: **v**
+- Mandatory
+- Permitted values: "txtv0"
+- Example: "v=txtv0"
 
-* Key: **to**
-* Mandatory
-* Permitted values: "absolute URL"
-* Example: "to=https://example.com/subpage/"
-<!--root/website?-->
+**Upstream URL**
+
+- Key: **to**
+- Mandatory
+- Permitted values: "absolute URL"
+- Example: "to=https://example.com/subpage/"
+  <!--root/website?-->
```

Comment:
Do we need a root/website? That was still an open question, but I think with a proxy we would most likely not need this as all failures should be handled by the upstream. If you agree, we can remove this comment.